net/grpc/gateway/docker/node_interop_server/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM grpcweb/common
+FROM node:12-stretch
```

Comment:
As discussed, perhaps let's lock the version here to `node:12.22.6-stretch` to match what we have in the `selenium` image below.

Reply:
Very good idea! Done! :)